Prace_domowe/Praca_domowa2/Adrian Stańdo/pd_02_finished.ipynb

```diff
@@ -0,0 +1,3118 @@
+{
```

Comment:
<p>ad. Wady:</p><ul><li>w jaki sposób zależność&nbsp;od rozkładu targetu jest wadą?</li><li>Dlaczego niejednoznaczność&nbsp;to problem? Może coś w kontekście oerfittingu?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>